tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -289,67 +288,157 @@ final class GlobalBuildRunner extends Runner {
     return result.ok;
   }
 
-  // TODO(zanderso): This should start and stop RBE when it is an --rbe build.
-  Future<bool> _runNinja(RunnerEventHandler eventHandler) async {
-    final String ninjaPath = p.join(
-      engineSrcDir.path, 'flutter', 'third_party', 'ninja', 'ninja',
+  late final String _hostCpu = (){
```

Comment:
You might (here and below) be able to get away with a regular `final`

Reply:
Tried this. Since here and below the initializers access instance members like `platform`, they can't be regular `final`.